examples/scripts/reward_modeling.py

```diff
@@ -100,27 +135,32 @@ def preprocess_function(examples):
 
         return new_examples
 
-    # Preprocess the dataset and filter out examples that are longer than args.max_length
-    # Compute that only on the main process for faster data processing.
-    # see: https://github.com/huggingface/trl/pull/1255
     with PartialState().local_main_process_first():
+        chosen_fn = conversations_formatting_function(tokenizer, "chosen")
```

Comment:
This assumes the `chosen` column is in the messages format `[{"role": "user", "content": "foo"} ...]` which I think is fine since that's almost the default standard